<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SceneWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="leftLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="frontLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="bottomLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
   </widget>
   <widget class="QLabel" name="topLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="QLabel" name="backLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneWidget</class>
   <extends>QWidget</extends>
   <header>SceneWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
